be/src/vec/columns/column_string.h

```diff
@@ -62,13 +62,13 @@ class ColumnStr final : public COWHelper<IColumn, ColumnStr<T>> {
 
     static constexpr size_t MAX_STRINGS_OVERFLOW_SIZE = 128;
 
-    void static check_chars_length(size_t total_length, size_t element_number) {
+    void static check_chars_length(size_t total_length, size_t element_number, size_t rows = 0) {
```

Comment:
change to not static

Reply:
so many caller in Doris eg: `ColumnString::check_chars_length`